control_panel_api/tests/permissions/test_tooldeployment_permissions.py

```diff
@@ -0,0 +1,55 @@
+from unittest.mock import MagicMock, patch
+
+from model_mommy import mommy
+from rest_framework.reverse import reverse
+from rest_framework.status import (
+    HTTP_201_CREATED,
+    HTTP_403_FORBIDDEN,
+)
+from rest_framework.test import APITestCase
+
+
+class ToolDeploymentPermissionsTest(APITestCase):
```

Comment:
Don't know if it's worth rename this file (as you may have moved on) `test_tooldeployment_permissions.py` => `test_tool_deployment_permissions.py`